<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PreferencesDialog</class>
 <widget class="QMainWindow" name="PreferencesDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>487</width>
    <height>476</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Preferences</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources/icons.qrc">
    <normaloff>:/icons/logo.png</normaloff>:/icons/logo.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
    background-color: #FFFFFF;
}</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>6</number>
      </property>
      <item>
       <widget class="QGroupBox" name="output_GroupBox">
        <property name="title">
         <string>Output</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <property name="bottomMargin">
          <number>12</number>
         </property>
         <property name="verticalSpacing">
          <number>-1</number>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="suffix_Label">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Suffix</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0" colspan="2">
          <widget class="QLineEdit" name="outputFolder_LineEdit">
           <property name="placeholderText">
            <string/>
           </property>
          </widget>
         </item>
         <item row="3" column="0" colspan="3">
          <widget class="QCheckBox" name="skip_CheckBox">
           <property name="text">
            <string>Skip if output size is greater than original</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QPushButton" name="outputFolderBrowse_Button">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Browse...</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1" colspan="2">
          <widget class="QLineEdit" name="suffix_LineEdit">
           <property name="placeholderText">
            <string>_compressed</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0" colspan="3">
          <widget class="QCheckBox" name="sameInputFolder_CheckBox">
           <property name="text">
            <string>Same folder as input</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="leftMargin">
         <number>6</number>
        </property>
        <property name="rightMargin">
         <number>6</number>
        </property>
        <property name="bottomMargin">
         <number>12</number>
        </property>
        <item>
         <widget class="QCheckBox" name="overwrite_CheckBox">
          <property name="styleSheet">
           <string notr="true">QCheckBox{border-left:5px;}</string>
          </property>
          <property name="text">
           <string>Overwrite the original file</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="warning_Spacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="warning_Button">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="font">
           <font>
            <underline>true</underline>
           </font>
          </property>
          <property name="text">
           <string>Original files will be lost</string>
          </property>
          <property name="icon">
           <iconset resource="../resources/icons.qrc">
            <normaloff>:/icons/warning.png</normaloff>:/icons/warning.png</iconset>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="importing_GroupBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string>Importing</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="spacing">
          <number>-1</number>
         </property>
         <property name="topMargin">
          <number>12</number>
         </property>
         <property name="bottomMargin">
          <number>12</number>
         </property>
         <item>
          <widget class="QCheckBox" name="scanSubfolders_CheckBox">
           <property name="text">
            <string>Scan subfolder while importing</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="interface_GroupBox">
        <property name="title">
         <string>Interface</string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="1">
          <widget class="QComboBox" name="locale_ComboBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <item>
            <property name="text">
             <string>-- System Language --</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="locale_Label">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Language</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <spacer name="locale_Spacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="0" colspan="3">
          <widget class="QCheckBox" name="promptBeforeExit_CheckBox">
           <property name="text">
            <string>Prompt before exit</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>-1</number>
      </property>
      <item>
       <spacer name="accept_Spacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="accept_Button">
        <property name="text">
         <string>Accept</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolBar {
background-color: #FFFFFF;
border-bottom: 1px solid #F8F9FA;
}</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>true</bool>
   </attribute>
   <addaction name="actionGeneral"/>
  </widget>
  <action name="actionGeneral">
   <property name="icon">
    <iconset resource="../resources/icons.qrc">
     <normaloff>:/icons/general.png</normaloff>:/icons/general.png</iconset>
   </property>
   <property name="text">
    <string>General</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../resources/icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>overwrite_CheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>output_GroupBox</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>70</x>
     <y>236</y>
    </hint>
    <hint type="destinationlabel">
     <x>129</x>
     <y>109</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sameInputFolder_CheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>outputFolder_LineEdit</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>83</x>
     <y>173</y>
    </hint>
    <hint type="destinationlabel">
     <x>203</x>
     <y>145</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sameInputFolder_CheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>outputFolderBrowse_Button</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>352</x>
     <y>170</y>
    </hint>
    <hint type="destinationlabel">
     <x>389</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>accept_Button</sender>
   <signal>clicked()</signal>
   <receiver>PreferencesDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>416</x>
     <y>445</y>
    </hint>
    <hint type="destinationlabel">
     <x>191</x>
     <y>441</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
